docs: email verified behaviour when using the Facebook provider (#5148)
FirebaseExtended/flutterfire: docs/auth/social.mdx

</TabItem>
</Tabs>

:::caution
Firebase will not set the `User.emailVerified` property

> to `true` if your user logs in with Facebook. The user's Facebook provider will be overridden should your user login using a provider that verifies email(e.g. Google).

Reviewer: ```suggestion
to `true` if your user logs in with Facebook. Should your user login using a provider that verifies email (e.g. Google sign-in) then this will be set to true.
```
